backend/dataall/modules/dataset_sharing/services/share_item_service.py

```diff
@@ -128,15 +173,9 @@ def add_shared_item(uri: str, data: dict = None):
                     itemName=item.name,
                     status=ShareItemStatus.PendingApproval.value,
                     owner=context.username,
-                    GlueDatabaseName=dataset.GlueDatabaseName
-                    if item_type == ShareableType.Table.value
-                    else '',
-                    GlueTableName=item.GlueTableName
-                    if item_type == ShareableType.Table.value
-                    else '',
-                    S3AccessPointName=s3_access_point_name
-                    if item_type == ShareableType.StorageLocation.value
-                    else '',
+                    GlueDatabaseName=dataset.GlueDatabaseName if item_type == ShareableType.Table.value else "",
+                    GlueTableName=item.GlueTableName if item_type == ShareableType.Table.value else "",
+                    S3AccessPointName=s3_access_point_name if item_type == ShareableType.StorageLocation.value else "",
                 )
```

Comment:
When we add a share_item we are not setting the `healthStatus` and the other health columns' values. Because `healthStatus` is defined as `healthStatus = Column(String, nullable=True, default=ShareItemHealthStatus.Healthy.value)` in `backend/dataall/modules/dataset_sharing/db/share_object_models.py` As a result the share item starts in PendingApproval and in Healthy healthStatus, but I think you wanted to start in None right?

Reply:
good catch! - used to have default as such but meant to change back to allow start as null, will fix